packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -2,7 +2,11 @@ import React, { useState, useEffect, useCallback } from 'react';
 import debounce from 'lodash/debounce';
 
 // leafygreen-ui
-import Icon from '@leafygreen-ui/icon';
+import BuildingIcon from '@leafygreen-ui/icon/dist/Building';
+import CaretUpIcon from '@leafygreen-ui/icon/dist/CaretUp';
+import CaretDownIcon from '@leafygreen-ui/icon/dist/CaretDown';
+import SettingsIcon from '@leafygreen-ui/icon/dist/Settings';
+import { Size } from '@leafygreen-ui/icon';
```

Comment:
This might not work how we hope... I have a sinking feeling that this might include the full bundle still. There is a silver lining – I think you might be able to import this from `dist` as well now.

Reply:
Since we have typescript hinting, we could just use the string, `'small'` like we were before as well.

Reply:
👍